spikeinterface/extractors/neoextractors/spikeglx.py

```diff
@@ -49,6 +50,11 @@ def __init__(self, folder_path, stream_id=None):
             else:
                 self.set_probe(probe, in_place=True)
             self.set_probe(probe, in_place=True)
+            
+            # load sample shifts
+            imDatPrb_type = probe.annotations["imDatPrb_type"]
+            sample_shifts = _get_sample_shifts(self.get_num_channels(), imDatPrb_type)
+            self.set_property("inter_sample_shift", sample_shifts)
```

Comment:
The actual master already contain tis no ?
I don't understand.

Reply:
Is it a bug from github formattting

Reply:
Yes, it is not showing on my diff.